content/components/selectpanel.mdx

```diff
@@ -9,51 +9,156 @@ import {Box} from '@primer/react'
 <img
   width="960"
   alt="Select panel for assigning labels to an issue. 'Apply labels to this issue' heading, close button in upper right hand corner. Search input with a magnifer icon as a visible label, listbox of label options and the first three are selected with a visual checkbox. The footer contains a button to edit labels, and cancel and apply buttons. Purple labels are overlayed onto select panel: Overlay (dialog) pointing to the container, Search input pointing to the input, role='listbox' bracket of the listbox content, Optional action control to the button in the lower left corner, Explicit confirm / submission buttons bracket for the apply and cancel buttons in the lower right corner."
-  src="https://user-images.githubusercontent.com/40274682/229016794-4620a95e-efa8-486d-a413-fdadafaa33a0.png"
+  src="https://github.com/primer/design/assets/980622/6e4b27cc-7a90-4958-af26-791c23e02b7d"
 />
 
+The select panel is composed of three main areas: a header, a scrollable list, and an optional footer.
+
+- **Header:** The header always includes a title and a close button. Additionally, it can have an optional search/filter field to find items quicker. When a selection is made, a "clear all" button should also be present.
+- **List (scrollable):** The scrollable list consists of [action list](/components/action-list) items, which can be either single or multi-select options.
+- **Footer (Optional):** In most cases, especially in multi-select scenarios, the select panel includes a footer that incorporates a "Cancel" and an "Apply" button also refered to as declaritive actions. If any additional controls are required, such as an extra checkbox or button, a footer becomes necessary as well.
+
 ## Options
 
+The select panel provides flexibility by offering both simple list-based single selection and multi-selection options. Additionally, it can become more sophisticated by incorporating advanced features such as filtering functionality and the inclusion of links.
+
 <img
   width="960"
   alt="First select panel is a branch switcher control that includes tabs, a search input, a listbox with single-select items, and a footer with an All branches link, and cancel and apply buttons. Second select panel is titled 'Show fields, with a list of about 10 multi-select items and apply and cancel buttons in the footer."
-  src="https://user-images.githubusercontent.com/40274682/229011161-abc2d548-fcf1-4bed-b5a2-eb88d2121a09.png"
+  src="https://github.com/primer/design/assets/980622/17c26b4f-e4ac-4120-8714-1bfffc2b73da"
 />
-<Caption>Select panel can have as much as the branch selector component, or as little as just a multi-selection list. If the contents request user input beyond a semantic menu(example: the context menu on comments in GitHub), it’s likely you need a select panel.</Caption>
 
-### Additional control slot
+### Trigger
+
+<img width="960" alt="" src="https://github.com/primer/design/assets/980622/f6a1e670-2682-4ce7-8642-24943084c812" />
```

Comment:
Should this say "action menu item"? It was a little unclear for me at first.

Reply:
Good spot! Fixed the acton menu item copy in both the image and markdown.

@ichelsea Do you have an example where we call them `inline labels` in the docs? I couldn't find any references of this after doing a search.

The only argument I could come up with for using `internal` over `inline` is that `inline` often has it's own meaning in CSS.

Right now[ in action menu it's also called external labels](https://primer.style/design/components/action-menu#trigger).